gcp/kubernetes/README.md

```diff
@@ -1,29 +1,43 @@
-# Kubernetes Configuration
+# jiant on GKE
```

Comment:
Insane nit (you're allowed to ignore): jiant => `jiant` in all running text.

Reply:
Updated.